src/services/Task/TaskReview/TaskReviewNeeded.js

```diff
@@ -43,14 +43,16 @@ export const fetchReviewNeededTasks = function(criteria, limit=50) {
                                                           criteria.boundingBox,
                                                           _get(criteria, 'savedChallengesOnly'),
                                                           _get(criteria, 'excludeOtherReviewers'))
+  const includeTags = criteria.includeTags
 
   return function(dispatch) {
     return new Endpoint(
       api.tasks.review,
       {
         schema: {tasks: [taskSchema()]},
         variables: {},
-        params: {limit, sort, order, page: (page * limit), ...searchParameters},
+        params: {limit, sort, order, page: (page * limit), ...searchParameters,
```

Comment:
Is `page * limit` still correct? If this touches refactored server code with the new Paging mechanism, it may not be.

Reply:
It still uses the old server code.